RobotArdu/src/main/java/de/fhg/iais/roberta/worker/compile/ArduinoCompilerWorker.java

```diff
@@ -110,6 +119,21 @@ public void execute(Project project) {
         String sourceDir = tempDir + token + "/" + programName + "/source/";
         String targetDir = tempDir + token + "/" + programName + "/target/";
 
+        String[] executableWithParametersWin =
```

Comment:
This is not needed. My suggestion is to use the old declaration (Line 137) and add in case of Windows the two parameters to `executableWithParameters`

Reply:
create string array with list